camp/mgetachew/mgetachewMiniProject/src/main/java/frc/robot/subsystems/MotorTalon.java

```diff
@@ -0,0 +1,85 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix6.configs.Slot0Configs;
+import com.ctre.phoenix6.StatusCode;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.VelocityVoltage;
+import com.ctre.phoenix6.controls.VoltageOut;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import com.ctre.phoenix6.hardware.TalonFX;
+import frc.robot.Constants;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+
+public class MotorTalon extends SubsystemBase{
+    //configurtition for the motor begins here
+    private TalonFX motorTalon;
+    //velcity and voltage control
+    private final VelocityVoltage motorTalonVelocityController = new VelocityVoltage(0);
+    private final VoltageOut motorTalonVoltageController = new VoltageOut(0);
+    private TalonFXConfiguration motorTalonMotorConfiguration = null;
+    private Slot0Configs motorRpmGains = new Slot0Configs().withKS(0.5);
+
+    private int reverseSpeedRpm = -1;
+    private int forwardSpeedRpm = 1;
+    private NeutralModeValue motorTargetNeutralModeValue = NeutralModeValue.Coast;
+    private static final double kMinDeadband = 0.001;
+      
+    //construct motor and print out the fact that its done.
+    public MotorTalon(){
+        motorTalon = new TalonFX(Constants.OperatorConstants.motorTalonPort);
+        configureMotor();
+        System.out.print("Motor is set");
+        motorTalonVelocityController.withSlot(0);
+    }
+
+    public void motorReverse(){
+        //reverse motor
+        motorTalon.setControl(this.motorTalonVelocityController.withVelocity(reverseSpeedRpm));
```

Comment:
I'm not sure the code works as written. Remember on the driver station laptop, we implemented a class variable, requestedSpeed, that was set by each of these motorReverse, motorForward, motorStop methods, and then in periodic, we set the value on the motor in an if-else statement. We also implemented a previousSpeed and tested whether the previous speed was the same as the requested speed and then only changed it on the motor if it was different. That may be required in order to fix the stutter. 
We can test this as-is and then add the code back if this doesn't work as you have written it here.